raiden/tests/integration/test_transfers.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+import pytest
+import gevent
+
+from raiden.tests.utils.transfer import channel
+from raiden.tests.utils.network import CHAIN
+
+
+@pytest.mark.parametrize('privatekey_seed', ['event_new_channel:{}'])
+@pytest.mark.parametrize('number_of_nodes', [4])
+@pytest.mark.parametrize('channels_per_node', [CHAIN])
+@pytest.mark.parametrize('settle_timeout', [50])
+@pytest.mark.parametrize('reveal_timeout', [5])
+def test_identifier_collision(raiden_chain, token_addresses, deposit):
```

Comment:
Appreciate the test! But it has nothing to do with the PR, which was to clarify some comments only. Could have easily been its own PR and its own commit.